### PR TITLE
multi feature dev

### PR DESCRIPTION
**Pull request by dodo4513**
hi

my name is do young

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -48,8 +48,7 @@ int sub(int number1, int number2)
  
 int multi(int number1, int number2)
 {
-	//TODO implementation
-	return 0;
+	return number1*number2;
```
Comment (dodo4513) [inline main.c:51]: very good!

**Review by dodo4513 — COMMENTED** [review]
very good!!

File: main.c (hunk 1 above)
Comment (HyeonGuJ) [reply main.c:51]: :+1:  good!!

Comment (HyeonGuJ) [inline main.c]: good!!

Comment (dedenzzi) [reply main.c:51]: :+1:

Comment (coffeeminus) [reply main.c:51]: good :seedling: :+1:

**Comment by dodo4513** [comment]
💯

**Comment by dodo4513** [comment]
ㅇㅇㅇ

**Pull request merged**